gradle.properties

```diff
@@ -19,10 +19,11 @@ microutilsLoggingVersion = 2.0.11
 apacheHttpClientVersion = 5.2.3
 aesyDatasizeVersion = 1.0.0
 
-sharedLibsRef = merge/v0.8.6
+sharedLibsRef = feature/fix-gradle-scripts-macos-EPMDJ-10786
 sharedLibsLocalPath = lib-jvm-shared
 nativeAgentLibName = drill-agent
 nativeAgentHookEnabled = false
+macosLd64 = false

```

Comment:
What about using it in java-agent module?

Reply:
added